spotify.html

```diff
@@ -6,42 +6,53 @@
     <link rel="stylesheet" href="body.css">
+    <link rel="stylesheet" href="footer.css">
+    <link rel="stylesheet" href="header.js">
+    <script src="header.js"></script>
 </head>
 <body>
     <header class="header">
-        <div class="left-header">
-            <div class="img-div">
-                <img class="img-logo" src="img/Spotify_Logo_RGB_White.png">
+        <nav class="navbar">
+            <a href="#" class="nav-logo">WebDev.</a>
+            <ul class="nav-menu">
+                <li class="nav-item">
+                    <a href="#" class="nav-link">Services</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link">Blog</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link">About</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link">Contact</a>
+                </li>
+            </ul>
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
             </div>
-        </div>
-       
-        <div class="right-header">
-            <div></div>
-            <div class="premium"><p>Premium</p></div>
-            <div class="support"><p>Support</p></div>
-            <div class="download"><p>Download</p></div>
-            <div class="slash">|</div>
-            <div class="profile-pic"><img class="profile-icon"  src="img/profile icons.png"></div>
-            <div class="profile"><p class="profile-text">Profile</p></div>
-        </div>
-    </header>
-  
-    <div class="line1">
-
-        <div class="text">
-            
-                <h1>Get more out of your music with Premium Individual.</h1>
-                <h2>Enjoy uninterrupted music listening with Spotify Premium.</h2>
-                <h2>Starting at only NGN 900. Cancel anytime.</h2>
-                <div>
-                <button class="btn1">Get Premium Individual</button> <button class="btn2">View Premium plans</button>
-                </div>
+        </nav>
+</header>
 
-                <u><h4>Terms apply.</h4></u>
-          </div>
-
-            <img src=https://wwwmarketing.scdn.co/static/images/premium/desktop-album-evergreen-1x.png>            
-        </div>        
+    
+    <div class="line1">
+        <div class="text">            
+            <h1>Get more out of your music with Premium Individual.</h1>
+            <h2>Enjoy uninterrupted music listening with Spotify Premium.</h2>
+            <h2>Starting at only NGN 900. Cancel anytime.</h2>
+            <div>
+            <button class="btn1">Get Premium Individual</button> <button class="btn2">View Premium plans</button>
+            </div>
+            <u><h4>Terms apply.</h4></u>
+      </div>
+      <img src="https://wwwmarketing.scdn.co/static/images/premium/desktop-album-evergreen-1x.png">  
```

Comment:
it's good practice you alt attributes to your images `<img src="image-source"  alt="some-description" >`